src/App.tsx

```diff
@@ -15,36 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  byAlphabet = 'byAlphabet',
+  byLength = 'byLength',
+  byDefault = 'byDefault',
+}
+
+const getPreparedGoods = (
+  goods: string[],
+  sortField?: SortType | undefined,
+  reverse?: boolean,
+) => {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    switch (sortField) {
+      case SortType.byAlphabet:
+        preparedGoods.sort((el1, el2) => el1.localeCompare(el2));
```

Comment:
```suggestion
        preparedGoods.sort((a, b) => a.localeCompare(b));
```
or firstGood and secondGood